src/Catalyst.Core.Modules.Dfs/CoreApi/ObjectApi.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Core.Modules.Dfs.UnixFileSystem;
+using Lib.P2P;
+
+namespace Catalyst.Core.Modules.Dfs.CoreApi
+{
+    class ObjectApi : IObjectApi
+    {
+        internal static IDagNode EmptyNode;
+        internal static IDagNode EmptyDirectory;
+
+        IDfs ipfs;
+
+        static ObjectApi()
+        {
+            EmptyNode = new DagNode(new byte[0]);
+            var _ = EmptyNode.Id;
+
+            var dm = new DataMessage
+            {
+                Type = DataType.Directory
+            };
+            using (var pb = new MemoryStream())
+            {
+                ProtoBuf.Serializer.Serialize<DataMessage>(pb, dm);
+                EmptyDirectory = new DagNode(pb.ToArray());
+            }
+
+            _ = EmptyDirectory.Id;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable '_'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)